README.md

```diff
@@ -5,6 +5,65 @@ This repository is aimed to be added as git submodule into particular
 containers' source repositories.  By default, the path to submodule should be
 named 'common'.
 
+Usage
+-----
+
+This section explains the usage of the shared scripts in this repository when
+it is used as a submodule in a container source repository.
+
+Once you have the repository set as a submodule include `common.mk` into your
+root Makefile in order to access the default rules used to call shared scripts.
+
+**Default rules:**
+
+`make` or `make build`  
+This rule will build an image without tagging it with any tags after it is built.
```

Comment:
Add info about squashing and need to have installed `docker-squash` (https://github.com/goldmann/docker-squash) tool?

Reply:
IMO this is implementation detail ... so I wouldn't insist on that.  RHEL images will get
squashed nad centos not.  Documentation would be fine, but I don't think it belongs to
this paragraph.

Reply:
Right, will add a note, thanks